BigIntegerTests/Operators.cs

```diff
@@ -456,6 +456,7 @@ public void TestDecrement()
         [TestMethod]
         public void TestShiftRight()
         {
+            (new BigInteger()).ToHexString();
```

Comment:
Sorry about that. I wanted to test the method but have forgotten to delete the line.